samples/project-anfield/import_cluster_config.py

```diff
@@ -559,6 +604,58 @@ def create_sources(source, environment, node):
                 body.password = password
             if env == "view":
                 body.view_type = view_enum.KVIEWBOX
+
+        elif environment == env_enum.K_HYPERV:
+            prot_source = source.protection_source
```

Comment:
We need to break this into more modular functions . lets add a TODO or techdebt ticket for later